lib/scryptenc/scryptenc.c

```diff
@@ -241,10 +242,25 @@ scryptenc_setup(uint8_t header[96], uint8_t dk[64],
 		/* Check logN, r, p. */
 		if ((rc = checkparams(P->maxmem, P->maxmemfrac, P->maxtime,
 		    P->logN, P->r, P->p, verbose, force)) != 0) {
+			/* Warn about resource limit, but suppress the error. */
+			if (rc == SCRYPT_ETOOBIG) {
+				warn0("Explicit parameters require more memory "
+				    "than is available; continuing anyway.");
```

Comment:
Rather than "... continuing anyway", how about "Warning: Explicit parameters may require too much memory" (and similarly for ETOOSLOW below)?  It's possible that someone will use explicit parameters on systems where the memlimit code doesn't work, so I think the "may" is meaningful.

Reply:
If I did
```
$ scrypt enc --STUFF foo bar.enc
scrypt: Warning: Explicit parameters may require too much memory
```

and then nothing happened for a while, I'd assume that the program was in an infinite loop or was waiting for user input for some reason.  So I'd hit <enter> a few times, then ctrl-c.

I'm happy to add the "may" (or "might") in there, but I think we should have some language to indicate that the process is continuing.